examples/custom-server/README.md

```diff
@@ -0,0 +1,116 @@
+# Payload Custom Server Example
+
+This example demonstrates how to serve your front-end alongside [Payload](https://github.com/payloadcms/payload) in a single Express server. This pattern will cut down on hosting costs and can simplify your deployment process.
+
+## Quick Start
+
+To spin up this example locally, follow these steps:
+
+1. First clone the repo
+1. Then `cd YOUR_PROJECT_REPO && cp .env.example .env`
+1. Next `yarn && yarn dev`
+1. Now `open http://localhost:3000/admin` to access the admin panel
+1. Login with email `dev@payloadcms.com` and password `test`
+
+That's it! Changes made in `./src` will be reflected in your app. See the [Development](#development) section for more details.
+
+## How it works
+
+When you use Payload, you plug it into _**your**_ Express server. That's a fundamental difference between Payload and other application frameworks. It means that when you use Payload, you're technically _adding_ Payload to _your_ app, and not building a "Payload app".
+
+One of the strengths of this pattern is that it lets you do powerful things like integrate your Payload instance directly with your front-end. This will allow you to host Payload alongside a fully dynamic, CMS-integrated website or app on a single, combined server—while still getting all of the benefits of a headless CMS.
+
+### Express
+
+In every Payload app is a `server.ts` file in which you instantiate your own Express server and attach Payload to it. This is where you can can add any custom Express middleware or routes you need to serve your front-end. To combine Payload with your framework on the same server, we need to do three main things:
+
+1. Modify your `server.ts` file to build and serve your front-end using the APIs provided by your framework
+2. Modify your `package.json` scripts include your framework's build commands
+3. Use a separate `tsconfig.server.json` file to build the server, because your front-end may require incompatible TypeScript settings
+
+This example demonstrates how to do this with [Next.js](https://nextjs.org), although the same principles apply to any front-end framework like [Vue](https://vuejs.org), [Nuxt](https://nuxt.com), or [SvelteKit](https://kit.svelte.dev). If your framework does not yet have instructions listed here, please consider contributing them to this example for others to use. To quickly eject Next.js from this example, see the [Eject](#eject) section.
+
+#### Next.js
+
+For Next.js apps, your `server.ts` file looks something like this:
+
+```ts
+import next from 'next'
+import nextBuild from 'next/dist/build'
+
+// Instantiate Express and Payload
+// ...
+
+// If building, start the server to build the Next.js app then exit
+if (process.env.NEXT_BUILD) {
+  app.listen(3000, async () => {
+    await nextBuild(path.join(__dirname, '../'))
+    process.exit()
+  })
+}
```

Comment:
Is this code still relevant? When building using `next build` this file is not run.
```suggestion
```

Reply:
See note below!